### ISSUE TITLE
Please create a new version

### ISSUE THREAD

**Issue by roumano** [GRIF-IRFU/puppet-omd#39]
Actually puppet only see your first version (v1.0.3) (3 year old) :  
https://forge.puppet.com/fschaer/omd/readme

And the master is 18 commit behind this version...
It's will permit to everyone to used the last version automatically.

**Comment by fschaer** [comment]
Yes, I will release updates - but there were very few changes though.
